Mathlib/Analysis/NormedSpace/Completion.lean

```diff
@@ -111,6 +112,25 @@ instance [SeminormedCommRing A] [NormedAlgebra 𝕜 A] [UniformContinuousConstSM
         simp only [← coe_smul, norm_coe]
         exact norm_smul_le r x }
 
+instance instNormedField [NormedField A] [CompletableTopField A] :
+    NormedField (UniformSpace.Completion A) where
+  toField := UniformSpace.Completion.instField
+  dist_eq := fun x y => by
+      refine Completion.induction_on₂ x y ?_ ?_ <;> clear x y
+      · refine isClosed_eq (Completion.uniformContinuous_extension₂ _).continuous ?_
+        exact Continuous.comp Completion.continuous_extension continuous_sub
+      · intro x y
+        rw [← Completion.coe_sub, norm_coe, Completion.dist_eq, dist_eq_norm]
```

Comment:
```suggestion
  dist_eq x y := by
    refine Completion.induction_on₂ x y ?_ ?_ <;> clear x y
    · refine isClosed_eq (Completion.uniformContinuous_extension₂ _).continuous ?_
      exact Continuous.comp Completion.continuous_extension continuous_sub
    · intro x y
      rw [← Completion.coe_sub, norm_coe, Completion.dist_eq, dist_eq_norm]
```

Reply:
Fixed, and also for the `norm_mul'` field